packages/ns-reverse-proxy/Makefile

```diff
@@ -1,41 +0,0 @@
-#
```

Comment:
If we are going to merge the packages, we should remove the old one from the build config: https://github.com/NethServer/nethsecurity/blob/main/config/ns-reverse-proxy.conf

Reply:
Was sure something like this was needed, what are the steps?

Reply:
Just remove the `ns-reverse-proxy.conf` file from the `config` dir.